substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/DefaultOptionHandler.java

```diff
@@ -151,6 +152,18 @@ public boolean consume(ArgumentQueue args) {
                 args.poll();
                 NativeImage.showWarning("Ignoring server-mode native-image argument " + headArg + ".");
                 return true;
+            case "--exclude-config":
+                args.poll();
+                String excludeConfig = args.poll();
+                if (excludeConfig == null) {
+                    NativeImage.showError(headArg + " requires a jar and resource regular expression, comma separated");
+                }
+                final String[] split = excludeConfig.split(",");
```

Comment:
If your regex contains a `","` you will only use the first half of it as the second argument to nativeImage.addExcludeConfig. What you need here is
```java
String[] split = excludeConfig.split(",", 2);
```